phpunit.xml.dist

```diff
@@ -2,15 +2,19 @@
 
 <phpunit
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/5.7/phpunit.xsd"
```

Comment:
How about referencing the schema by a relative URL? 

```diff
-xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/5.7/phpunit.xsd"
+xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
```

This way it will always be accurate!